src/components/Post/Tabs/PostOnChainInfo/index.tsx

```diff
@@ -105,11 +105,11 @@ export const getBlockNumber = (statusHistory?: {
 const PostOnChainInfo: FC<IPostOnChainInfoProps> = (props) => {
 	const { network } = useNetworkContext();
 
-	const { className, handleOpenSidebar, onChainInfo, proposalType } = props;
```

Comment:
remove handleOpenSidebar from props and also from Post.tsx where this component getting called

Reply:
resolved, pls check